pages/linux/ncat.md

```diff
@@ -9,3 +9,7 @@
 - Write output of specified file to the specified host on the specified port:
 
 `ncat {{address}} {{port}} < {{/path/to/file}}`
+
+- Accept multiple connections and keep them open after they have been closed:
```

Comment:
What is meant by "keep them open after they have been closed"? Why would I want it to do that?

Just curious.

Reply:
The wording is incorrect here. It is not the _connection_ that remains open, rather _ncat_ itself which is open.

This section from the man page will clarify - 

```
-k, --keep-open (Accept multiple connections) .
           Normally a listening server accepts only one connection and then quits when the connection is closed. This option makes it
           accept multiple simultaneous connections and wait for more connections after they have all been closed. It must be combined with
           --listen. In this mode there is no way for Ncat to know when its network input is finished, so it will keep running until
           interrupted. This also means that it will never close its output stream, so any program reading from Ncat and looking for
           end-of-file will also hang.
```

Reply:
Ah, I see.